replay-scripts/src/main/python/replay/resources/replay_config.py

```diff
@@ -0,0 +1,175 @@
+def init():
+    return {
+        "ingest": {
+            "outputKafkaTopic": "telemetry.ingest",
+            "inputPrefix": "ingest",
+            "dependentSinkSources": [
+                {
+                    "type": "azure",
+                    "prefix": "raw"
+                },
+                {
+                    "type": "azure",
+                    "prefix": "unique"
+                },
+                {
+                    "type": "azure",
+                    "prefix": "channel"
+                },
+                {
+                    "type": "azure",
+                    "prefix": "telemetry-denormalized/raw"
+                },
+                {
+                    "type": "druid",
+                    "prefix": "telemetry-events"
+                },
+                {
+                    "type": "druid",
+                    "prefix": "telemetry-log-events"
+                },
+                {
+                    "type": "druid",
+                    "prefix": "telemetry-error-events"
+                },
+                {
+                    "type": "druid",
+                    "prefix": "telemetry-feedback-events"
```

Comment:
Account for the rollup and syntcs data sources. Also can the prefix be an array to simplify the configuration?

Reply:
also add a variable for druid sink sources on the indexing date - syncts or ets. For syncts the segments delete is straight forward and for ets it is based on postgres lookup